src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -262,6 +267,21 @@ bool SendLocationRequest::CheckHMICapabilities(
   return true;
 }
 
+bool SendLocationRequest::CheckHMINaviCapabilities() {
```

Comment:
@LitvinenkoIra function name is not correct

Reply:
@LuxoftAKutsan 901166817624f7fdce93816e544293a37ee20c65